### DISCUSSION TITLE
Render Markdown in all (Card/Grid/Standard/Journal) views

### DISCUSSION THREAD

**Discussion by aisbergde** [GitJournal/GitJournal#804] [Feature Requests]
it is fine to start the edit of notes in markdown mode or in raw mode, but when notes are listed or shown I wan't to see them rendered, like in the preview, when I edit entries. There should be a switch between plain text and preview mode.

Currently I see only plain text, no matter what I select
- Standard View
- Journal View
- Grid View
- Card View

Version Info:
GitJournal 1.54.0+1355 (Google Play Store Version)

---
> Please vote on this issue with a 👍 reaction. It's one of the primary mechanisms used to prioritize issues. Please also subscribe to the issue if you're interested, as that way I can discuss possible implementations and share mockups.

**Comment by vHanda** [comment]
Hey Germo.

In the Markdown editor you can toggle to show it in preview mode. Is that
not working for you?

**Comment by aisbergde** [comment]
I know I can do this in the editor per note. But what I request is to show the preview when I use and browse notes, better: to be able to switch between plain text and rendered text. 

For example I want to switch from google keep to a markdown based notes system (you have another app to export google keep into markdown, but there is an issue). And I want to *use* the notes system as a "normal" notes system, showing formatted text.

I like the idea to use git for my markdown notes system very much. And it looks like I can also put .txt data into git and it is not shown by GitJournal. Very good, I could keep markdown notes and todo.txt in one place as I do now, but I could use git instead of OneDrive and I would have a version control. 

BTW, I can't find where the files are located on android, I search be filename but I can't find. And it is not documented. Maybe you could document or even better show this information directly in the app?

**Comment by vHanda** [comment]
Ah. Now I understand. Yeah I should really render the notes in at least the grid or cards view. Will do.

Currently GitJournal only supports files which ends in '.md'. Let me add this to the FAQ. Also the notes are currently stored in a directory only accessible by the app. I'm actively working on allowing you to move it. Gimme some time.

**Comment by aisbergde** [comment]
You are on the right way. But currently I still need to use my OneNote solution if I can't access the data with other apps.

You wrote about a FAQ, but here https://github.com/GitJournal/GitJournal is no link to any FAQ, and also the WIKI doesn't exist. I was looking for a FAQ but did not find. If there is a FAQ please promote the link.

now I searched and I found: https://gitjournal.io/support/ but the link should also be on the project landing page.
